### PR TITLE
Use make rules from kubernetes/kubernetes.

### PR DESCRIPTION
**Pull request by spxtr**
Companion to https://github.com/kubernetes/kubernetes/pull/39252. I removed the failure reason because make eats exit codes.

### CODE REVIEW

**Comment by spxtr** [comment]
@mikedanese okay ready to go.

File: jobs/pull-kubernetes-bazel.sh [hunk 1]
```diff
@@ -20,29 +20,19 @@ set -o pipefail
...
 if [[ "${rc}" == 0 ]]; then
-  bazel test  --test_output=errors --test_tag_filters '-skip' //cmd/... //pkg/... //federation/... //plugin/... //build/... //third_party/... && rc=$? || rc=$?
+  make bazel-test && rc=$? || rc=$?
 fi
 
 if [[ "${rc}" == 0 ]]; then
   bazel run //:ci-artifacts -- "gs://kubernetes-release-dev/bazel/$(git rev-parse HEAD)" && rc=$? || rc=$?
 fi
 
-case "${rc}" in
```
Comment (mikedanese) [inline jobs/pull-kubernetes-bazel.sh]: why delete this?

**Comment by spxtr** [comment]
> I removed the failure reason because make eats exit codes.

**Pull request merged**